wasp/apps/settings.py

```diff
@@ -115,6 +123,11 @@ def _draw(self):
             self._dd.draw()
             draw.set_font(fonts.sans24)
             draw.string('DD    MM    YY',0,180, width=240)
+        elif self._current_setting == 'Notifications':
+            self._von.state = wasp.system.vibe_on_notif
+            self._won.state = wasp.system.wake_on_notif
+            self._von.draw()
+            self._won.draw()
```

Comment:
It might be a good idea to add the new checkbox to the existing page (with the slider) and update the title.